test/test_t8code_2d.jl

```diff
@@ -30,7 +30,19 @@ mkdir(outdir)
     end
 end
 
+@trixi_testset "test check_for_negative_volumes" begin
+    # Unstructured mesh with six cells which have left-handed node ordering.
+    mesh_file = joinpath(EXAMPLES_DIR, "rectangle_with_negative_volumes.msh")
+    isfile(mesh_file) ||
+        download("https://gist.githubusercontent.com/jmark/bfe0d45f8e369298d6cc637733819013/raw/cecf86edecc736e8b3e06e354c494b2052d41f7a/rectangle_with_negative_volumes.msh",
+                 mesh_file)
+
+    # This call should throw a warning about negative volumes detected.
+    mesh = T8codeMesh(mesh_file, 2)
```

Comment:
Should negative volumes yield just a warning or even an error?

Reply:
According to this: https://docs.julialang.org/en/v1/stdlib/Logging/ I think `@warn` fits best. The simulation technically runs "normally" besides having instability issues.

Here is an excerpt:

- Logging.Debug (log level -1000)
`is information intended for the developer of the program. These events are disabled by default.`
- Logging.Info(log level 0)
`is for general information to the user. Think of it as an alternative to using println directly.`
- Logging.Warn(log level 1000)
`means something is wrong and action is likely required but that for now the program is still working.`
- Logging.Error (log level 2000)
`means something is wrong and it is unlikely to be recovered, at least by this part of the code. Often this log-level is unneeded as throwing an exception can convey all the required information.`

Reply:
Okay 👍